
# Details of a button
string button_id
# bounding box vertices in screen coordinates
# nav_2d_msgs/Point2D[] button_polygon

# button center in global coordinates
geometry_msgs/Transform button_transform 

# bounding box vertices in local coordinates
float64 x0
float64 y0
float64 x1
float64 y1

